config/default.json

```
@@ -9,7 +9,7 @@
         "git": {
           "path": "./data/versions",
           "publish": false,
-          "prefixMessageToSnapshotId": "This version was recorded after filtering snapshot in repository ./data/snapshots with commit ",
+          "snapshotURI": "./data/snapshots/%SNAPSHOT_ID",
```

This is not a URI but a file path. Either use the `file://` scheme or start using the [`x-git-object`](http://www.nuke24.net/docs/2014/GitObjectURIScheme.html) scheme.

```suggestion
          "snapshotURI": "file://./data/snapshots/%SNAPSHOT_ID",
```